sequencing/ATACSeq/preprocessing/fragmentDistribution.r

```diff
@@ -28,32 +41,61 @@ calcNucleoProps<-function(fraglen.stand){
 	return(c(freeSum, monoSum, diSum, triSum, otherSum))
 }
 
+fdsPlots <- function(listSamples){
+  counter = 1
+  list_plots <- vector("list", 10)
+  samples <- ls(listSamples)
+  for(i in 1:length(listSamples)) {
+    df <-data.frame(listSamples[i])
+    colnames(df)<- c("Var1", "Freq")
+    df$sample <-samples[i]
+    list_plots[[i]] <- df
+    counter = counter + 1
+  }
+  bigdf <-ldply(list_plots, data.frame)
+  bigdf[,1]<- as.integer(as.character(bigdf[,1]))
+  return(bigdf)
+}
+
+## ==========##
+##   SET-UP  ##
+## ==========##
+
 args <- commandArgs()
-alignedDir<-args[6]
+project <-args[6]
+source(paste0("/lustre/projects/Research_Project-MRC190311/ATACSeq/",project,"/config.r")
```

Comment:
Why not just pass the location of the config.r file into the command line? The bash script that calls this Rscript already knows where the config.r file is, so it could just pass it into this script right? That would make this less complex and also means there is no need to paste a full file path.